zh-cn: update the translation of the learn/css index page (#23446)
mdn/translated-content: files/zh-cn/learn/css/index.md

{{LearnSidebar}}

层叠样式表——或 {{glossary("CSS")}}——是你在学完 {{glossary("HTML")}} 之后应该学习的第一门技术。HTML 用于为你的内容定义结构和语义，CSS 用于为内容提供样式和对内容进行布局。例如，你可以使用 CSS 更改内容的字体、颜色、大小和间距，将内容分成多列，或添加动画和其它装饰特性。

## 前提

在尝试 CSS 之前，你应该学习 HTML 的基础知识。我们建议你先学习 [介绍 HTML](/zh-CN/docs/Learn/HTML/Introduction_to_HTML)。

> 一旦你理解了 HTML 的基础知识，我们就建议你同时学习额外的 HTML 和 CSS，在这两个专题之间来回切换。这么做是因为，使用 CSS 后，HTML 会更有意思，学习 HTML 也更有趣。并且不了解 HTML 的话，也无法学习 CSS。

Author: 后续的修改，会遵循这样的原则“没有变得比之前的表达好，就不会选择修改”。

> 在开始本专题之前，你还应该熟悉如何使用电脑，如何被动地使用 Web（例如，只是浏览 Web、消费 Web 上的内容）。你应该按照[安装基础软件](/zh-CN/docs/Learn/Getting_started_with_the_web/Installing_basic_software)中的详细内容搭建基础工作环境，以及按照[处理文件](/zh-CN/docs/Learn/Getting_started_with_the_web/Dealing_with_files)中的内容理解如何创建和管理文件——两个都是 [Web 入门](/zh-CN/docs/Learn/Getting_started_with_the_web)纯新手模块的一部分。

Reviewer: 前面已经有 Web 了，没必要再加（原翻译即可）

```suggestion
在开始本专题之前，你还应该熟悉如何使用电脑，如何被动地使用 Web（例如，只是浏览和消费内容）。你应该按照[安装基础软件](/zh-CN/docs/Learn/Getting_started_with_the_web/Installing_basic_software)中的详细内容搭建基础工作环境，以及按照[处理文件](/zh-CN/docs/Learn/Getting_started_with_the_web/Dealing_with_files)中的内容理解如何创建和管理文件——两个都是 [Web 入门](/zh-CN/docs/Learn/Getting_started_with_the_web)纯新手模块的一部分。
```

在开始本专题之前，还是建议你先学习 [Web 入门](/zh-CN/docs/Learn/Getting_started_with_the_web)，尤其当你是 Web 开发的纯新手的话。不过，[CSS 基础](/zh-CN/docs/Learn/Getting_started_with_the_web/CSS_basics)这篇文章中涉及的大部分内容在 [CSS 第一步](/zh-CN/docs/Learn/CSS/First_steps)模块中也有所涉及，而且更详细。

## 模块

> 本专题包含以下模块，按照建议的顺序学习。你应该从第一模块开始学习。

Reviewer: ```suggestion
本专题包含以下模块，建议按顺序学习这些模块。你应该从第一个模块开始。
```

- [CSS 第一步](/zh-CN/docs/Learn/CSS/First_steps)

>   - : CSS（层叠样式表）用于为网页添加样式，对网页进行布局——例如，修改内容的字体、颜色、大小和间距，将内容分成多列，或添加动画和其它装饰特性。在前往掌握 CSS 基础知识（CSS 的运作方式，CSS 语法是什么样子，在 HTML 中如何使用 CSS 添加样式）的道路上，本模块是一个很好的开始。

Reviewer: 看一下最后一句话原先的翻译，我感觉原来的翻译稍微改一下会更好

Author: 这一段是从 https://developer.mozilla.org/zh-CN/docs/Web/Tutorials#%E5%88%9D%E7%BA%A7_2 这里复制过来的。 https://developer.mozilla.org/zh-CN/docs/Web/Tutorials#%E5%88%9D%E7%BA%A7_2 的内容是前面刚修改过的。

- [CSS 构建块](/zh-CN/docs/Learn/CSS/Building_Blocks)

>   - : 本模块承接 [CSS 入门](/zh-CN/docs/Learn/CSS/First_steps)——这时你已经熟悉 CSS 语言及其语法、有了基本的使用经验，是时候再深入一些。在本模块中，你会学习层叠和继承、可用的选择器类型、单位、大小、为背景和边框添加样式、调试，以及其它内容。

Reviewer: - 前后文的一致性
- 第二处应该是用逗号好一点（带动词的短语）

```suggestion
  - : 本模块承接 [CSS 第一步](/zh-CN/docs/Learn/CSS/First_steps)——这时你已经熟悉 CSS 语言及其语法，有了基本的使用经验，是时候再深入一些。在本模块中，你会学习层叠和继承、可用的选择器类型、单位、大小、为背景和边框添加样式、调试，以及其它内容。
```

Author: 这一段也是从 https://developer.mozilla.org/zh-CN/docs/Web/Tutorials#%E5%88%9D%E7%BA%A7_2 复制过来的（也有相同的错误存在）

    本文的目的是，在了解诸如[为文本添加样式](/zh-CN/docs/Learn/CSS/Styling_text)和 [CSS 布局](/zh-CN/docs/Learn/CSS/CSS_layout)等特定领域的知识之前，提供一个能让你编写合格 CSS 的工具集并帮助你理解基本的理论。

- [为文本添加样式](/zh-CN/docs/Learn/CSS/Styling_text)
  - : 在学完 CSS 语言的基础知识后，下一个集中学习的 CSS 主题是为文本添加样式——用 CSS 做的最多的事之一。本文介绍了为文本添加样式的基础知识，包括：设置字体、加粗、斜体、行间距、字间距、阴影和其它文本特性。在本模块中，还会学习的内容有：在页面中应用自定义字体、为列表和链接添加样式。
- [CSS 布局](/zh-CN/docs/Learn/CSS/CSS_layout)

>   - : 看到这儿，你已经学习过 CSS 基础知识、如何为文本添加样式、如何为放置内容的盒子添加样式并操纵它。现在，是时候学习参照视口或另一个盒子时如何将盒子放置在正确的位置。必要的前置知识已经具备，此时你可以深入 CSS 布局，学习不同的显示设置、现代的布局工具（如弹性盒、CSS 网格）、定位、以及一些你可能仍想了解的传统技术。

Reviewer: ```suggestion
  - : 看到这儿，你已经学习过 CSS 基础知识、如何为文本添加样式、如何为放置内容的盒子添加样式并操纵它。现在，是时候学习参照视口或另一个盒子时如何将盒子放置在正确的位置。必要的前置知识已经具备，此时你可以深入 CSS 布局，学习不同的显示设置、现代的布局工具（如弹性盒、CSS 网格）、定位，以及一些你可能仍想了解的传统技术。
```

## 解决常见的 CSS 问题

> [使用 CSS 解决常见问题](/zh-CN/docs/Learn/CSS/Howto)提供的链接中的内容用于解释如何使用 CSS 解决创建网页时常遇到的问题。

Reviewer: ```suggestion
[使用 CSS 解决常见问题](/zh-CN/docs/Learn/CSS/Howto)提供了用于解释如何使用 CSS 解决创建网页时常遇到的问题的内容的链接。
```
